src/QsFmt/App/Program.fs

```diff
@@ -14,11 +14,11 @@ open Microsoft.Quantum.QsFmt.Formatter
 let makeFullPath input =
     if input = "-" then input else Path.GetFullPath input
 
-let run (commandWithOptions: CommandWithOptions) inputs =
+let runCommand (commandWithOptions: CommandWithOptions) inputs =
 
     let mutable paths = Set.empty
 
-    let rec doOne input =
+    let rec runOneCommand input =
```

Comment:
nit: The nested functions are really operating on inputs rather than commands. It is not as though `runManyCommands` actually runs multiple _commands_. It processes multiple inputs, so maybe the names could be something like `doOneInput` and `doManyInputs`?